src/pivot-table/components/cells/utils/get-dimension-cell-style.ts

```diff
@@ -108,13 +109,15 @@ export const getContainerStyle = ({
     ...resolvedLockedSelectionStyle,
     display: "flex",
     zIndex,
+    justifyContent: isLeftColumn ? undefined : "center",
```

Comment:
you can do the same thing here: 
```suggestion
    ...(!isLeftColumn && { justifyContent: "center" })
```